backend/src/modules/idim-webservice/idim-webservice.controller.spec.ts

```diff
@@ -69,4 +69,54 @@ describe('IdimWebserviceController', () => {
             ).rejects.toThrowError('Http Exception');
         });
     });
+
+    describe.skip('verifyBceidUser', () => {
```

Comment:
I wrote the test cases, but skip them for now, cause currently we need to use IDIR GUID to search for bceid user, and I don't feel comfortable to use a real IDIR GUID to do the test, and post it public. So later we can use the test bceid guid to do this.